src/__test__/Details.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import Details from '../pages/Details';
+
+const mockStore = configureMockStore([]);
+
+const initialState = {
+  country: {
+    data: [
+      {
+        name: 'Test Country',
+        flag: 'test-flag-url',
+        officialName: 'Official Test Country',
```

Comment:
- Can you work on the test again? because it could not pass.
- 
![me](https://github.com/siddghosh108/React-Country-Capstone/assets/63972670/480f6adf-73fc-47e8-bfcb-e19424a89069)
-
![Capture](https://github.com/siddghosh108/React-Country-Capstone/assets/63972670/d05662bd-4cd1-41f8-b8d3-92fb8b1fa3e7)